drivers/vfio/vfio.c

```diff
@@ -827,31 +770,29 @@ int vfio_add_group_dev(struct device *dev,
 		iommu_group_put(iommu_group);
 	}
 
-	device = vfio_group_get_device(group, dev);
-	if (device) {
-		dev_WARN(dev, "Device already exists on group %d\n",
+	existing_device = vfio_group_get_device(group, device->dev);
```

Comment:
Here the name is as I'd expect "vfio_group" prefix and the action - "get_device"